Feature branch - Cloned sign up page of mint.com

Rumbie-Mudzie7 opened the pull request:
In this project, we cloned a replica of the mint.com sign up page.  The project has equipped us with the necessary skills to create a professional model of a web sign up page.

Rumbie-Mudzie7 committed:
initial code commit

Rumbie-Mudzie7 committed:
added stickler file and stylelint

ebeagusamuel committed:
Added the html structure for the form and some css styling

Rumbie-Mudzie7 committed:
Modified and added more styling

ebeagusamuel committed:
readme file modified

ebeagusamuel committed:
added a screenshot of the cloned sing up page

mikenath223 commented:
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 🐱

mikenath223 reviewed (changes requested):
Awesome Job Team 👏👏

You've done well on this, 🎉
Your project is almost at mastery level, just a few changes need to be made to move forward.👌

**Kindly fix this issues:👇**

- [ ] Required - Please enable [stickler](https://stickler-ci.com/) on your repo, and fix the linting issues.

- [ ] Required - Kindly avoid using inline styling in your HTML code, consider moving all styles to your stylesheet. Avoid basic CSS bad practices refer to this ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)`

- [ ] Required - Add a project live link to your Readme, consider using this [tool](https://raw.githack.com/) to create a live link.

- [ ] Required - Please make use of semantic HTML tags(`main`,`header`,`section`,`footer`. etc) in your code. Refer to this [link](https://www.w3schools.com/html/html5_semantic_elements.asp) on semantic elements. Also kindly ensure they are "appropriately sized and positioned" (as described in Odin project description) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/semantic_html_requirement.png) from Pathwright)`

- [ ] Required - There are aspects of the original webpage  that are clickable, but these are not clickable links in yours👇

* The intuit logo and the three images below it are meant to be clickable.👇[Screenshot]
![ice_screenshot_20200330-185520](https://user-images.githubusercontent.com/33323077/77946662-99eab200-72ba-11ea-9370-daec8cf8810d.png)

* Next the form submit button is meant to be clickable👇[Screenshot]
![ice_screenshot_20200330-185457](https://user-images.githubusercontent.com/33323077/77946723-b555bd00-72ba-11ea-934d-ba39ac21b7df.png)

* Also the `Sign in` button located at the top right-hand corner is supposed to be clickable:👇[Screenshot]
![ice_screenshot_20200330-185533](https://user-images.githubusercontent.com/33323077/77946814-dc13f380-72ba-11ea-8677-c651266e3911.png)

**Once more the frontend of your project looks awesome 👌.**
Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

Rumbie-Mudzie7 committed:
stylelint error solved

Rumbie-Mudzie7 committed:
Live demo of project-mint.com included

We inserted a live demo of the project based on the requirements from the code review feedback

Rumbie-Mudzie7 committed:
Code formatting using html semantic elements

Rumbie-Mudzie7 committed:
Merge branch 'feature-branch' of https://github.com/Rumbie-Mudzie7/project-mint.com into feature-branch

majovanilla commented on index.html:
```diff
@@ -0,0 +1,114 @@
+        <li><a href="#">Privacy</a></li>
+        <li><a href="#">Security</a></li>
+      </ol>
+      <p style="padding-bottom: 5px;">
```
Please remove inline styling. It's not considered a good practice.

majovanilla commented on index.html:
```diff
@@ -0,0 +1,114 @@
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav class="sign-in">
+      <div>
+        <p>Already have an Intuit account?</p>
+        <a href="#">I forgot my User ID or password</a>
+      </div>
+      <button><i class="fas fa-lock"></i> Sign In</button>
+    </nav>
```
Nice use of semantic HTML as requested

majovanilla commented on README.md:
```diff
@@ -1,2 +1,55 @@
+
+![screenshot](./img/Screenshot1.png)
+
+# Live demo 
+[Live demo](https://raw.githack.com/Rumbie-Mudzie7/project-mint.com/feature-branch/index.html)
+
```
Great! You added the link as suggested.

majovanilla reviewed (changes requested):
Hey team, 👋

## Good job implementing the required changes on this project! 👍 you have some more left!!

- You didn't activate the linters and we can't see the checks yet. Please make this happen.
- Check the commented code below.

- Always remember to double (or triple) check to see if you made ALL the changes required in the previous review, this will save a lot of time for you!


## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

Rumbie-Mudzie7 committed:
removed inline styling

ebeagusamuel committed:
made some styling changes to the css reset file

Rumbie-Mudzie7 committed:
Formatted reset.css file and made fixes

Rumbie-Mudzie7 committed:
fixes on reset.css

ebeagusamuel committed:
reset.css modifications made

ebeagusamuel committed:
corrected lint error on style.css file

ebeagusamuel committed:
corrected identation error

mikenath223 reviewed (changes requested):
### Wonderful Job here Team🎉🎉

**There has been a major uplift in your project👍👍💪**

However, there exists just one👌 minimal error that prevents us from moving forward. You are almost there💪

- [ ] Required - Kindly parse your HTML through this validator [tool](https://validator.w3.org/nu/) to ensure your HTML meets the w3c standards. Fix the issue referenced there.

**You've got this team💪🎉🎉**

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻🎉

ebeagusamuel committed:
made some validation correction

Dandush03 commented:
### :heavy_check_mark: Project Approve :boom:  :checkered_flag:
# Hey @Rumbie-Mudzie7 :hand: 
### Great Job so Far :grey_exclamation: :grey_exclamation: :muscle: :smiley: 

### Following Previous Reviews and Seeing you Have Made Required Changes. 
### The project Looks Great.:grey_exclamation::wink:

### Good Luck in your Next Assigment
### Happy coding:computer: 
#### Daniel Laloush
